webapp/components/wpt-bsf.js

```diff
@@ -61,6 +61,7 @@ class WPTBSF extends LoadingState(PolymerElement) {
       </style>
       <div class="bsf">
         <div class="left">
+          <h5>Browser-specific failures are the number of WPT tests which fail in exactly one browser. This graph shows the BSF scores for Chrome, Firefox and Safari over the last two years.</h5>
```

Comment:
I don't think `over the last two years` is particularly useful information. Let's drop that last part (and just end on "This graph shows the BSF scores for Chrome, Firefox and Safari.").